src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java

```diff
@@ -280,6 +278,7 @@ public void siControllaCheLaDelegaHaLoStatoAttiva(String dpFile) {
         logger.info("Si controlla che la delega abbia lo stato Attiva");
         this.deleghe = this.dataPopulation.readDataPopulation(dpFile + ".yaml");
         this.leTueDelegheSection.controlloStatoAttiva(deleghe.get("name").toString(), deleghe.get("familyName").toString());
+        this.leTueDelegheSection.controlloStatoAttiva(deleghe.get("name").toString(), deleghe.get("familyName").toString());
```

Comment:
you do not need to `.toString()`, the `.get()` return the type of the value declared on the hashmap

Reply:
![image](https://github.com/pagopa/pn-frontend-e2e/assets/119851973/5e48dd76-6f7b-4512-a559-e572c77eebe6)